projects/api/src/product/entities/product.entity.ts

```diff
@@ -16,9 +16,24 @@ export default class ProductEntity implements Product {
   @ApiProperty()
   description: string;
 
+  @ApiProperty()
+  brand: string;
+
+  @ApiProperty()
+  image: string;
+
   @ApiProperty()
   price: number;
 
+  @ApiProperty()
+  stock: number;
```

Comment:
hmmm, How do you guys identify if a user enters an integer positive number? since javascript does not distinguish integer or double, it just use Number for both